charts/block-node-server/templates/deployment.yaml

```diff
@@ -41,6 +41,24 @@ spec:
             claimName: {{ if .Values.blockNode.persistence.live.create }}{{ include "hiero-block-node.fullname" . }}-live{{ else }}{{ .Values.blockNode.persistence.live.existingClaim }}{{ end }}
         - name: unverified-ephemeral-storage
           emptyDir: {}
+      initContainers:
+        - name: init-storage-dirs
+          image: busybox
+          command:
+            - sh
+            - -c
+            - |
+              mkdir -p /live-pvc/live-data && \
```

Comment:
nit: in future we might want to parametrize the names but this is good.
Do we have this paths captured in operational docs anywhere?
So an operator has a clear idea of what paths are of improtant.
Non blocking consideration